app/assets/javascripts/discourse/app/controllers/move-to-topic.js

```diff
@@ -110,6 +113,64 @@ export default Controller.extend(ModalFunctionality, {
     return canSplitTopic && this.currentUser && this.currentUser.admin;
   },
 
+  confirmMergeTypeDialog(isMessage) {
+    return new Promise((resolve, reject) => {
+      const dialogMessage = isMessage
+        ? I18n.t("topic.merge_topic.merge_type.title", {
+            count: this.selectedPosts.length,
+          })
+        : I18n.t("topic.move_to_existing_message.merge_type.title", {
+            count: this.selectedPosts.length,
+          });
+
+      const sequentialLabel = isMessage
+        ? I18n.t("topic.move_to_existing_message.merge_type.sequential")
+        : I18n.t("topic.merge_topic.merge_type.sequential");
+
+      const chronologicalLabel = isMessage
+        ? I18n.t("topic.move_to_existing_message.merge_type.chronological")
+        : I18n.t("topic.merge_topic.merge_type.chronological");
+
+      this.dialog.alert({
+        message: dialogMessage,
```

Comment:
Yeah, I tried it first but thought it could be a little too much:

![image](https://user-images.githubusercontent.com/3530/236255329-9350e97f-b7d2-4ba2-b7f0-6c06072a8491.png)